src/analysis/modals/ComputeModal/AzureComputeModal/AzureComputeModal.test.ts

```diff
@@ -399,20 +403,20 @@ describe('AzureComputeModal', () => {
   it('deletes disk when there is no runtime present.', async () => {
     // Arrange
     const user = userEvent.setup();
-
     const disk = getDisk();
+    const diskDeleteFn = jest.fn();
 
     const runtimeFunc = jest.fn(() => ({
       details: () => null,
     }));
-    Ajax.mockImplementation(() => ({
+    asMockedFn(Ajax).mockImplementation(() => ({
       ...defaultAjaxImpl,
       Runtimes: {
         runtime: runtimeFunc,
       },
       Disks: {
-        disk: () => ({
-          details: () => disk,
+        disksV2: () => ({
+          delete: diskDeleteFn,
```

Comment:
did this update the endpoint used to disksV2? or was our original test not actually testing what was there?

Reply:
The azure compute modal never actually calls the disk details endpoint mocked here, it gets its disks passed in from the parent. It was also never verifying that the delete endpoint was being called, just the visible display.